### PR TITLE
Update conditional-access-app-control-how-to-overview.md

### PR DESCRIPTION
**Pull request by damalkaw**
#sign off

### CODE REVIEW

File: CloudAppSecurityDocs/conditional-access-app-control-how-to-overview.md [hunk 1]
```diff
@@ -35,6 +35,10 @@ The following articles provide sample processes for configuring a non-Microsoft
 - [Active Directory Federation Services (AD FS) as your IdP](proxy-idp-adfs.md)
 - [Okta as your IdP](proxy-idp-okta.md)
 
+## Prerequisites: 
+1. Ensure your firewall configurations allow traffic from all the IP addresses listed [here](https://learn.microsoft.com/en-us/defender-cloud-apps/network-requirements)
```
Comment (batamig) [inline CloudAppSecurityDocs/conditional-access-app-control-how-to-overview.md:39]:
```suggestion
1. Ensure your firewall configurations allow traffic from all the IP addresses listed [here](network-requirements.md)
```

**Comment by Court72** [comment]
@batamig

**Important**: When the changes are ready for publication, adding a `#sign-off` comment is the best way to signal that the PR is ready for the review team to merge. 
	
#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: CloudAppSecurityDocs/conditional-access-app-control-how-to-overview.md [hunk 2]
```diff
@@ -35,6 +35,10 @@ The following articles provide sample processes for configuring a non-Microsoft
 - [Active Directory Federation Services (AD FS) as your IdP](proxy-idp-adfs.md)
 - [Okta as your IdP](proxy-idp-okta.md)
 
+## Prerequisites: 
+1. Ensure your firewall configurations allow traffic from all the IP addresses listed [here](network-requirements.md)
```
Comment (batamig) [inline CloudAppSecurityDocs/conditional-access-app-control-how-to-overview.md:39]:
```suggestion
1. Ensure your firewall configurations allow traffic from all the IP addresses listed [here](network-requirements.md).
```

**Comment by batamig** [comment]
#sign-off

File: CloudAppSecurityDocs/conditional-access-app-control-how-to-overview.md [hunk 3]
```diff
@@ -35,6 +35,10 @@ The following articles provide sample processes for configuring a non-Microsoft
 - [Active Directory Federation Services (AD FS) as your IdP](proxy-idp-adfs.md)
 - [Okta as your IdP](proxy-idp-okta.md)
 
+## Prerequisites: 
+1. Ensure your firewall configurations allow traffic from all the IP addresses listed [here](network-requirements.md).
+1. Confirm that your application possesses a complete certificate chain. Incomplete or partial certificate chains may lead to unexpected behavior in applications when monitored with Conditional Access app control policies.
```
Comment (Jak-MS) [inline CloudAppSecurityDocs/conditional-access-app-control-how-to-overview.md:40]:
```suggestion

1. Ensure your firewall configurations allow traffic from all the IP addresses listed on [Network requirements](network-requirements.md).
1. Confirm that your application possesses a complete certificate chain. Incomplete or partial certificate chains may lead to unexpected behavior in applications when monitored with Conditional Access app control policies.
```

**Pull request merged**